### PR TITLE
feat(curriculum): add English curriculum block 9.2 requesting and receiving guidance from others

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by miyaliu666** [comment]
I don't see why task-46 doesn't show "description" when running locally.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
One tiny nit

File: curriculum/challenges/_meta/learn-how-to-request-and-receive-guidance/meta.json [hunk 1]
```diff
@@ -9,8 +9,268 @@
   "superBlock": "a2-english-for-developers",
   "challengeOrder": [
     {
-      "id": "655cd6b40aabaf962afdf02d",
-      "title": "Dialogue: Placeholder"
+      "id": "66250fc0d711192eebfff0e9",
+      "title": "Dialogue 1:Solving Coding Challenges"
```
Comment (moT01) [inline curriculum/challenges/_meta/learn-how-to-request-and-receive-guidance/meta.json:13]:
```suggestion
      "title": "Dialogue 1: Solving Coding Challenges"
```

File: curriculum/challenges/_meta/learn-how-to-request-and-receive-guidance/meta.json [hunk 2]
```diff
@@ -9,8 +9,268 @@
...
+      "id": "66251e6701f1ac574db171fe",
+      "title": "Task 28"
+    },
+    {
+      "id": "66251f12b5f14c58d410cd98",
+      "title": "Task 29"
+    },
+    {
+      "id": "6625ceb55c430866094b40b9",
+      "title": "Dialogue 2: Learning A New Technology"
```
Comment (moT01) [inline curriculum/challenges/_meta/learn-how-to-request-and-receive-guidance/meta.json:133]:
```suggestion
      "title": "Dialogue 2: Learning a New Technology"
```

File: curriculum/challenges/_meta/learn-how-to-request-and-receive-guidance/meta.json [hunk 3]
```diff
@@ -9,8 +9,268 @@
...
+      "id": "6625dc1d103a638a7fd5308b",
+      "title": "Task 53"
+    },
+    {
+      "id": "6625dc81861c0d8b754a4829",
+      "title": "Task 54"
+    },
+    {
+      "id": "6625ddee54d1db9090a4800f",
+      "title": "Dialogue 3: Offering Assistance And Receiving Guidance"
```
Comment (moT01) [inline curriculum/challenges/_meta/learn-how-to-request-and-receive-guidance/meta.json:237]:
```suggestion
      "title": "Dialogue 3: Offering Assistance and Receiving Guidance"
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/66250fc0d711192eebfff0e9.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+id: 66250fc0d711192eebfff0e9
+title: "Dialogue 1: Solving Coding Challenges"
+challengeType: 21
+dashedName: dialogue-2-solving-coding-challenges
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/66250fc0d711192eebfff0e9.md:5]:
```suggestion
dashedName: dialogue-1-solving-coding-challenges
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625ceb55c430866094b40b9.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+id: 6625ceb55c430866094b40b9
+title: "Dialogue 2: Learning A New Technology"
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625ceb55c430866094b40b9.md:3]:
```suggestion
title: "Dialogue 2: Learning a New Technology"
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625ddee54d1db9090a4800f.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+id: 6625ddee54d1db9090a4800f
+title: "Dialogue 3: Offering Assistance And Receiving Guidance"
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625ddee54d1db9090a4800f.md:3]:
```suggestion
title: "Dialogue 3: Offering Assistance and Receiving Guidance"
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625d84e92201f802eac3973.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+challengeType: 22
+dashedName: task-46
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: When the documentation is extensive, start with the introductory sections.
+-->
+
+# --description–-
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-request-and-receive-guidance/6625d84e92201f802eac3973.md:13]:
```suggestion
# --description--
```

One of the dashes wasn't a regular dash or something - so the description wasn't showing up.

**Review by moT01 — CHANGES_REQUESTED** [review]
Looks good @miyaliu666 🎉 

Found a few things...

**Review by moT01 — APPROVED** [review]

**Review by nieldakarla — CHANGES_REQUESTED** [review]
This block looks great. Just made 2 adjustments to some descriptions.

**Review by nieldakarla — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Sorry for blocking this one too. 😔

**Pull request merged**